### PR TITLE
Update flashing-rpi.md

### PR DESCRIPTION
**Pull request by jh80chung**
Remove the unsupported guidance, and update the latest image URL.

### CODE REVIEW

File: docs/platform/developing/flashing-rpi.md [hunk 1]
```diff
@@ -451,20 +278,14 @@ To configure the Raspberry Pi board, follow the steps below:
 1.  Connect Smart Development Bridge (SDB) as described in [Connect the board to the PC](#connect-the-board-to-the-pc).
 2.  Install the connectivity drivers for each board:
     -   **Raspberry Pi 3/4**
-        1.  Download the plugin zip file from the **Raspberry Pi 3/4(6.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
+        1.  Download the plugin zip file from the **Raspberry Pi 3/4(7.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
```
Comment (omar-srbd) [inline docs/platform/developing/flashing-rpi.md:281]:
```suggestion
        1.  Download the plugin zip file from the **Raspberry Pi 3/4(7.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware>{:target="_blank"} and follow the instructions.
```

**Review by omar-srbd — COMMENTED** [review]
Hello @jh80chung, I have reviewed the file in this PR and left a question along with some correction suggestions.
Please look into the suggestions and incorporate them.
Thank you.

File: docs/platform/developing/flashing-rpi.md [hunk 2]
```diff
@@ -1,50 +1,29 @@
...
-> In Windows, only EXT4 type file of rootfs.img is supported to flash RPM bundles. If you are using Headless 32-bit and Headed 64-bit image, RPMs cannot be installed through IoT Setup Manager because rootsfs.img is of BTRFS type file.
-
 ## Prerequisites
 
 ### Operating system
 
-IoT Setup Manager supports the following operating systems:
-
--   Windows 10
 -   Ubuntu 18.04 LTS and later
```
[the remark this answers is not in the document]
Comment (jh80chung) [reply docs/platform/developing/flashing-rpi.md:18]: > I have a doubt regarding here. The removed line mentioned here was "**IoT Setup Manager supports the following operating systems:**". So, shouldn't the **Ubuntu** version be removed as well?

Even though its script is also  supported on  older ubuntu version, I don't recommend to use "older Ubuntu version". So Ubuntu version is remained. There are some dependents with sfdisk or other commands.

File: docs/platform/developing/flashing-rpi.md [hunk 3]
```diff
@@ -1,50 +1,29 @@
...
-sudo apt-get install <package-name>
-```
 
 ### Download binaries
 
 You must have the supported binary images in your computer. To download the binary images, follow these steps:
 
-1. Download images from [Downloads](http://download.tizen.org/releases/milestone/tizen).
-2. Select **6.0-unified (or "unified" whichever is present) > latest > images > standard**.
+1. Download images from [Downloads](http://download.tizen.org/releases/milestone/TIZEN).
```
[the remark this answers is not in the document]
Comment (omar-srbd) [reply docs/platform/developing/flashing-rpi.md:25]: External links in a .md file opens in a new window

File: docs/platform/developing/flashing-rpi.md (hunk 1 above)
Comment (omar-srbd) [reply docs/platform/developing/flashing-rpi.md:281]: External links in a .md file opens in a new window

**Comment by omar-srbd** [comment]
Hello @jh80chung, thanks for incorporating the changes as per suggestion. However, it seems you have missed a couple of suggested changes. Links for the missed out comments are as follows:

- https://github.com/Samsung/tizen-docs/pull/1774#discussion_r1109580800
- https://github.com/Samsung/tizen-docs/pull/1774#discussion_r1109663591

File: docs/platform/developing/flashing-rpi.md (hunk 1 above)
Comment (jh80chung) [reply docs/platform/developing/flashing-rpi.md]: Updated. Thanks!

File: docs/platform/developing/flashing-rpi.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jh80chung) [reply docs/platform/developing/flashing-rpi.md]: > External links in a .md file opens in a new window

Updated. Thanks!

**Comment by jh80chung** [comment]
> Hello @jh80chung, thanks for incorporating the changes as per suggestion. However, it seems you have missed a couple of suggested changes. Links for the missed out comments are as follows:
> 
> * [Update flashing-rpi.md #1774 (comment)](https://github.com/Samsung/tizen-docs/pull/1774#discussion_r1109580800)
> * [Update flashing-rpi.md #1774 (comment)](https://github.com/Samsung/tizen-docs/pull/1774#discussion_r1109663591)

Sorry. I had updated commit as your commit suggestion.

https://github.com/Samsung/tizen-docs/pull/1774/commits/7acc556cabeda2d6911e2e23eb78540bb6f50e74#diff-8aa570afd23832c2c7fab901d5e04289d1448503a29f2afa7534cfb100cade16R281

Is there any other missed thing? If be, let me know, plz.

File: docs/platform/developing/flashing-rpi.md [hunk 4]
```diff
@@ -451,20 +278,14 @@ To configure the Raspberry Pi board, follow the steps below:
 1.  Connect Smart Development Bridge (SDB) as described in [Connect the board to the PC](#connect-the-board-to-the-pc).
 2.  Install the connectivity drivers for each board:
     -   **Raspberry Pi 3/4**
-        1.  Download the plugin zip file from the **Raspberry Pi 3/4(6.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
+        1.  Download the plugin zip file from the **Raspberry Pi 3/4(7.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware>{:target="_blank"} and follow the instructions.
```
Comment (omar-srbd) [inline docs/platform/developing/flashing-rpi.md:281]:
```suggestion
        1.  Download the plugin zip file from the **Raspberry Pi 3/4(7.0) Plugin** section at [Tizen Device Firmware](http://developer.samsung.com/tizendevice/firmware){:target="_blank"} and follow the instructions.
```

**Review by omar-srbd — COMMENTED** [review]
Hello @jh80chung, please refer to the suggested change and incorporate it at your earliest convenience.
Thank you.

File: docs/platform/developing/flashing-rpi.md (hunk 2 above)
Comment (omar-srbd) [inline docs/platform/developing/flashing-rpi.md:18]:
```suggestion
Ubuntu 18.04 LTS and later
```

File: docs/platform/developing/flashing-rpi.md [hunk 5]
```diff
@@ -1,50 +1,29 @@
...
-```
 
 ### Download binaries
 
 You must have the supported binary images in your computer. To download the binary images, follow these steps:
 
-1. Download images from [Downloads](http://download.tizen.org/releases/milestone/tizen).
-2. Select **6.0-unified (or "unified" whichever is present) > latest > images > standard**.
+1. Download images from [Downloads](http://download.tizen.org/releases/milestone/TIZEN){:target="_blank"}.
+2. Select **Tizen-7.0 (or "Tizen" whichever is present)> Tizen-7.0-Unified (or "Tizen-unified" whichever is present) > latest > images > standard**.
```
Comment (omar-srbd) [inline docs/platform/developing/flashing-rpi.md:26]:
```suggestion
2. Select **Tizen-7.0 (or "Tizen" whichever is present) > Tizen-7.0-Unified (or "Tizen-unified" whichever is present) > latest > images > standard**.
```

**Review by omar-srbd — COMMENTED** [review]
@jh80chung, please refer to the suggestions and implement them.
Thank you.

**Review by omar-srbd — APPROVED** [review]
@jh80chung, thanks for implementing all the changes as per suggestion.
LGTM.

@asm-foysal, I am approving this PR for merging.

**Comment by jh80chung** [comment]
@omar-srbd , @asm-foysal , Thanks for applied this. Can I ask one thing? does it need to request PR to live branch? Otherwise, do you sync live branch with master on your side? Because, some developer had referred with wrong flashing guide.

**Comment by omar-srbd** [comment]
Hello @jh80chung, about your query in https://github.com/Samsung/tizen-docs/pull/1774#issuecomment-1451043947, yes we update the live branch from our end. As you have raised a concern regarding the old document being followed by some developers, we will open a new PR to merge the master branch to live ASAP. (:
Thank you.

**Comment by jh80chung** [comment]
> Hello @jh80chung, about your query in [#1774 (comment)](https://github.com/Samsung/tizen-docs/pull/1774#issuecomment-1451043947), yes we update the live branch from our end. As you have raised a concern regarding the old document being followed by some developers, we will open a new PR to merge the master branch to live ASAP. (: Thank you.

Hi @omar-srbd , Thanks for sharing information. :)  Because I need to report this to Tizen PM and my boss.

**Comment by omar-srbd** [comment]
@jh80chung, please be noted that the live site of Tizen docs (https://docs.tizen.org/) has been updated.
Thank you.

**Pull request merged**